### PR TITLE
Release 0.13.3

### PR DESCRIPTION
**Pull request by h3xxit**
Patch release on top of 0.13.1. (0.13.2 was tagged and built but never merged or published — cubic's findings on this PR were fixed first, in #115 — so 0.13.3 supersedes it.)

- **A file's own access verbs are read even when its YAML is more than the subset parser knows.** A whole-document configuration file (`.tool`, or a kind an overlay registers) with a folded description, a literal block, a nested map, a quoted value or a flow sequence had its `owner:` / `read:` / `write:` silently dropped — a reviewed, merged grant did not exist, and the file was unreadable by the very principal it named. The subset parser still answers first; the full parser (failsafe schema, so every scalar keeps the text it was written as) is consulted only when it cannot. (#113, #115)
- A skill's version history is reachable again, from the same `⋯` menu a Knowledge page has.
- The file tree's right-click menu stays on screen, is placed again when the window resizes under it, and keeps its top margin at the window's edge.
- The empty state's opening suggestions no longer offer `access.md` files.

Tag `v0.13.3` is on this branch's head; the image build is running. Publishing to npm happens from `main` after this merges.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: packages/core-backend/src/modules/access-model/access-grammar.ts [hunk 1]
```diff
@@ -799,6 +800,31 @@ export function parseAccessFile(
...
+ * understand. Before this fallback that meant the file's `owner:` / `read:` /
+ * `write:` were silently dropped: a grant that was reviewed and merged did
+ * not exist, and the file was unreadable by the very principal it named.
+ * So on a subset failure the frontmatter is read by the full parser and only
+ * its top-level mapping is used — the verb keys are looked up exactly as
+ * before, everything else is ignored exactly as before.
+ */
+function ownEntriesRoot(frontmatter: string): unknown {
+  const subset = parseYamlSubset(frontmatter);
+  if (subset.ok) return subset.value;
```
[the remark this answers is not in the document]
Comment (h3xxit) [reply packages/core-backend/src/modules/access-model/access-grammar.ts]: Valid — fixed in #115: a verb value the subset parser passed through as flow/quoted syntax now sends the document to the full parser (with tests for a flow sequence, quoted scalars inline and in a block list).

File: packages/core-frontend/src/shared/components/usePointerMenuPosition.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  useLayoutEffect(() => {
+    const place = () => {
+      const el = ref.current;
+      if (!el) return;
+      const w = el.offsetWidth;
+      const h = el.offsetHeight;
+      const left = Math.max(MENU_MARGIN, Math.min(x, window.innerWidth - w - MENU_MARGIN));
+      let top: number;
+      if (y + h <= window.innerHeight - MENU_MARGIN) {
+        top = y;
```
[the remark this answers is not in the document]
Comment (h3xxit) [reply packages/core-frontend/src/shared/components/usePointerMenuPosition.ts]: Fixed in #115: the downward placement is clamped to `MENU_MARGIN` and re-checked for fit at the clamped position; a hook test covers the top-edge case.
